launcher/src/main.rs

```diff
@@ -548,6 +549,10 @@ fn shard_manager(
         shard_args.push(otlp_endpoint);
     }
 
+    // In case we use sliding window, we may ignore the sliding in flash for some backends depending on the parameter.
+    shard_args.push("--max-batch-prefill-tokens".to_string());
+    shard_args.push(max_batch_prefill_tokens.to_string());
```

Comment:
Shouldn't it be `--max-input-tokens` actually ? Windowing doesn't apply if seqlen > window but it's multiple queries with each their input lengths < windowing, no?